invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/communityRecordsSearch/CommunityRecordsSearchAppLayout.js

```diff
@@ -36,7 +36,7 @@ export const CommunityRecordsSearchAppLayout = ({ config, appName }) => {
               <Count
                 label={(cmp) => (
                   <Trans key="communityRecordsSearch" count={cmp}>
-                    {cmp} results found
+                    {cmp.toLocaleString("en-US")} results found
```

Comment:
please don't hardcode the en-US locales, there is a way to read the locale of the user